<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutContactDialog</class>
 <widget class="QDialog" name="AjoutContactDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter un contact</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>471</width>
     <height>301</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>62</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sexe :</string>
    </property>
    <property name="buddy">
     <cstring>sexCBox</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>21</y>
      <width>43</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Prénom :</string>
    </property>
    <property name="buddy">
     <cstring>firstnameInput</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstnameInput">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
   </widget>
   <widget class="QComboBox" name="sexCBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>64</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Homme</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Femme</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="birthdateLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Date de naissance :</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="birthdateInput">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>1900</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="minimumDate">
     <date>
      <year>1900</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="mailLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>E-mail :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mailInput">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>250</number>
    </property>
   </widget>
   <widget class="QLabel" name="companyLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom de l'entreprise :</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="companyInput">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QFrame" name="frameAddress">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>471</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Adresse :</string>
     </property>
     <property name="buddy">
      <cstring>addressInput</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="complementInput">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>250</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Code Postal :</string>
     </property>
     <property name="buddy">
      <cstring>postalcodeInput</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ville :</string>
     </property>
     <property name="buddy">
      <cstring>cityInput</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Complément :</string>
     </property>
     <property name="buddy">
      <cstring>complementInput</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="postalcodeInput">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="cityInput">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="addressInput">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>250</number>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lastnameInput">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>21</y>
      <width>28</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom :</string>
    </property>
    <property name="buddy">
     <cstring>lastnameInput</cstring>
    </property>
   </widget>
   <zorder>frameAddress</zorder>
   <zorder>label_3</zorder>
   <zorder>label_2</zorder>
   <zorder></zorder>
   <zorder>firstnameInput</zorder>
   <zorder>sexCBox</zorder>
   <zorder>birthdateLabel</zorder>
   <zorder>birthdateInput</zorder>
   <zorder>mailLabel</zorder>
   <zorder>mailInput</zorder>
   <zorder>companyLabel</zorder>
   <zorder>companyInput</zorder>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="privateRButton">
      <property name="text">
       <string>Contact Privé</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="professionnalRButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Contact Professionnel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
